packages/mobile/package.json

```diff
@@ -81,15 +81,15 @@
     "react-native": "^0.62.2",
     "react-native-android-open-settings": "^1.3.0",
     "react-native-bip39": "git://github.com/celo-org/react-native-bip39#0439f00",
-    "react-native-blind-threshold-bls": "https://github.com/celo-org/react-native-blind-threshold-bls#cc36392",
+    "react-native-blind-threshold-bls": "https://github.com/celo-org/react-native-blind-threshold-bls#e071514",
     "react-native-camera": "^3.23.1",
     "react-native-clock-sync": "^1.0.0",
     "react-native-config": "https://github.com/luggit/react-native-config#89a602b",
     "react-native-confirm-device-credentials": "git://github.com/celo-org/react-native-confirm-device-credentials#436afa9",
     "react-native-contacts": "https://github.com/celo-org/react-native-contacts#e473717",
     "react-native-device-info": "^5.5.6",
     "react-native-exit-app": "^1.1.0",
-    "react-native-fast-crypto": "git+https://github.com/celo-org/react-native-fast-crypto#3946f9",
+    "react-native-fast-crypto": "https://github.com/celo-org/react-native-fast-crypto#df1c171",
```

Comment:
No logical changes to this package. I just merged the branch we were using there to master to make things easier to follow.

Reply:
Okay, fixed the reference in bip39